mobile/www/index.html

```diff
@@ -25,6 +25,11 @@
         <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width">
         <link rel="stylesheet" type="text/css" href="css/index.css">
         <title>Calorie Counter</title>
+        <script>
+        setTimeout(function() {
+            $("#cameraInst").remove();
+            }, 2000);
```

Comment:
I like this for sure, but would it somehow be possible to take a picture before this disappears? You could add a similar event on-click of the camera icon if so - and possibly drop the delayed remove() until the user takes action first.

Reply:
@dolph that was just for 2sec but fixed it with onclick event in the dev5 branch commit